src/com/squareup/ideaplugin/dagger/ShowUsagesAction.java

```diff
@@ -72,34 +56,13 @@
 import com.intellij.psi.search.ProjectScope;
 import com.intellij.psi.search.PsiElementProcessor;
 import com.intellij.psi.search.SearchScope;
-import com.intellij.ui.ActiveComponent;
-import com.intellij.ui.InplaceButton;
-import com.intellij.ui.ScreenUtil;
-import com.intellij.ui.SpeedSearchBase;
-import com.intellij.ui.SpeedSearchComparator;
-import com.intellij.ui.TableScrollingUtil;
-import com.intellij.ui.TableUtil;
+import com.intellij.ui.*;
 import com.intellij.ui.awt.RelativePoint;
 import com.intellij.ui.popup.AbstractPopup;
 import com.intellij.ui.table.JBTable;
 import com.intellij.usageView.UsageViewBundle;
-import com.intellij.usages.PsiElementUsageTarget;
-import com.intellij.usages.Usage;
-import com.intellij.usages.UsageInfo2UsageAdapter;
-import com.intellij.usages.UsageInfoToUsageConverter;
-import com.intellij.usages.UsageTarget;
-import com.intellij.usages.UsageToPsiElementProvider;
-import com.intellij.usages.UsageView;
-import com.intellij.usages.UsageViewManager;
-import com.intellij.usages.UsageViewPresentation;
-import com.intellij.usages.UsageViewSettings;
-import com.intellij.usages.impl.GroupNode;
-import com.intellij.usages.impl.NullUsage;
-import com.intellij.usages.impl.UsageGroupingRuleProviderImpl;
-import com.intellij.usages.impl.UsageNode;
-import com.intellij.usages.impl.UsageViewImpl;
-import com.intellij.usages.impl.UsageViewManagerImpl;
-import com.intellij.usages.impl.UsageViewTreeModelBuilder;
+import com.intellij.usages.*;
+import com.intellij.usages.impl.*;
```

Comment:
make sure you're formatting with the [square code style](https://github.com/square/java-code-styles) so you dont get these star imports